### PR TITLE
Added a quick animation for hover state in inputs

### PR DESCRIPTION
**Pull request by aguscruiz**
Added a transition for box shadow and border in inputs

### CODE REVIEW

File: src/components/formItem/formItem.svelte [hunk 1]
```diff
@@ -240,6 +240,8 @@
     background: var(--background);
     box-shadow: var(--shadow);
     border: 1px solid var(--border-color);
+    transition: box-shadow 0.15s ease-in-out, border-color 0.15s ease-in-out;
```
Comment (fallaciousreasoning) [inline src/components/formItem/formItem.svelte:243]: I think in other places we use 0.12s or 0.2s depending on the animation. Probably not super important though

**Review by fallaciousreasoning — APPROVED** [review]

File: src/components/formItem/formItem.svelte (hunk 1 above)
Comment (AlanBreck) [reply src/components/formItem/formItem.svelte:243]: Might be worth creating a token for transitions.

**Review by AlanBreck — COMMENTED** [review]
Apart from @fallaciousreasoning's comment about consistency, this looks good.

**Comment by aguscruiz** [comment]
@AlanBreck I do have some presets for animations that we can add as tokens
https://www.figma.com/design/g8Z0q6TMPYDq6zXh9Y7LWD/%F0%9F%8C%8E-Universal?m=auto&t=3IMNIFI3ukRfbOE6-1

File: src/components/formItem/formItem.svelte (hunk 1 above)
Comment (aguscruiz) [reply src/components/formItem/formItem.svelte]: Nice, thanks, updated to 0.2

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**